Author: Florian Ragwitz <rafl@debian.org>
Description: add /etc/LatexMk to @rc_system_files
Index: latexmk-4.41/latexmk.pl
===================================================================
--- latexmk-4.41.orig/latexmk.pl
+++ latexmk-4.41/latexmk.pl
@@ -934,7 +934,8 @@ else {
     @rc_system_files = ();
     foreach ( 'LatexMk', 'latexmkrc' ) {
        push @rc_system_files,
-            ( "/opt/local/share/latexmk/$_", 
+            ( "/etc/$_",
+	      "/opt/local/share/latexmk/$_", 
               "/usr/local/share/latexmk/$_",
               "/usr/local/lib/latexmk/$_" );
     }
